sdks/node-sdk/src/Conversations.ts

```diff
@@ -202,4 +231,31 @@ export class Conversations {
   hmacKeys() {
     return this.#conversations.getHmacKeys();
   }
+
+  streamConsent(callback?: StreamCallback<Consent[]>) {
+    const asyncStream = new AsyncStream<Consent[]>();
+
+    const stream = this.#conversations.streamConsent((err, value) => {
+      asyncStream.callback(err, value);
+      callback?.(err, value);
+    });
+
+    asyncStream.onReturn = stream.end.bind(stream);
+
+    return asyncStream;
+  }
+
+  streamPreferences(callback?: StreamCallback<PreferenceUpdate>) {
+    const asyncStream = new AsyncStream<PreferenceUpdate>();
+
+    const stream = this.#conversations.streamPreferences((err, value) => {
+      // TODO: remove this once the node bindings type is updated
+      asyncStream.callback(err, value as unknown as PreferenceUpdate);
+      callback?.(err, value as unknown as PreferenceUpdate);
```

Comment:
I could be wrong but I don't believe the preference update actually streams the new hmac key I think it just streams that new hmac keys exist not the actual key. cc @codabrink

Reply:
i'm seeing these values in the stream:
```
{
  type: 'V',
  HmacKeyUpdate: {
    key: [
      146, 129, 165, 209, 164, 178,  11, 177,
       29,  71, 187,  25,  80,  21,  33,  62,
      227,  40, 130, 197, 163,  88, 231, 253,
      172, 180, 176,  80, 112, 130, 127, 121
    ]
  }
}
{
  type: 'V',
  HmacKeyUpdate: {
    key: [
       76, 252, 220, 176,  36,  62, 245, 190,
      220, 151, 172,  77, 232, 247,  46, 220,
        2,   2,  63,  21, 244, 255, 149,  41,
      130,  53, 126, 162,  97,  32,   0,  35
    ]
  }
}
```

Reply:
i'm assuming the `V` type is a current limitation of how the enum is serialized.